packages/table/src/table.tsx

```diff
@@ -790,16 +790,24 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         const tableBottom = this.grid.getCumulativeHeightAt(this.grid.numRows - 1);
         const tableRight = this.grid.getCumulativeWidthAt(this.grid.numCols - 1);
 
-        const nextScrollTop = (tableBottom < viewportRect.top + viewportRect.height)
+        let nextScrollTop = (tableBottom < viewportRect.top + viewportRect.height)
             // scroll the last row into view
             ? Math.max(0, tableBottom - viewportRect.height)
             : viewportRect.top;
 
-        const nextScrollLeft = (tableRight < viewportRect.left + viewportRect.width)
+        let nextScrollLeft = (tableRight < viewportRect.left + viewportRect.width)
             // scroll the last column into view
             ? Math.max(0, tableRight - viewportRect.width)
             : viewportRect.left;
 
+        // reset the scroll position when scrolling is disabled
```

Comment:
Gah, meant to revert this area of changes. This doesn't actually work (was trying to scroll to `(0,0)` if the scroll _became_ disabled (by reducing row/col sizes or by reducing the number of either). 

I'll investigate how to do this properly now, actually.